src/json-patch-duplex.js

```diff
@@ -91,11 +92,13 @@ var jsonpatch;
             return removed;
         },
         move: function (obj, key, document) {
-            var originalValue = getValueByPointer(document, this.path);
-            var newValue = getValueByPointer(document, this.from);
-            applyOperation(document, { op: "remove", path: this.from });
-            applyOperation(document, { op: "add", path: this.path, value: newValue });
-            return originalValue;
+            var overwrittenValue = getValueByPointer(document, this.path);
+            if (overwrittenValue) {
+                overwrittenValue = deepClone(overwrittenValue);
```

Comment:
This is how it originally behaved. In case the move **target** was overwritten we need to keep a copy of it to return it as removed. I can remove this but there is a spec for it and I thought it's necessary.